README.md

```diff
@@ -1,14 +1,21 @@
-# vehicle-commander-qt
+# message-simulator-qt
 
-The Vehicle Commander Qt demonstrates developer samples for building and testing in-vehicle military applications with the ArcGIS Runtime Qt SDK.  The Vehicle Commander Qt projects contains source code for creating sample in-vehicle, touchscreen, and support applications.
+Message Simulator provides UDP messaging for applications and system components in the ArcGIS for the Military solution, including:
 
-![Image of Vehicle Commander]( ScreenShot.jpg "vehicle-commander-qt")
+* [Vehicle Commander](https://github.com/Esri/vehicle-commander-java)
+* [Squad Leader](https://github.com/Esri/squad-leader-android)
+* Custom transports and adapters for ArcGIS GeoEvent Extension for Server ([solutions-geoevent-java](https://github.com/Esri/solutions-geoevent-java))
+
+Note: the Qt version of Vehicle Commander that was in this repo has been retired. If you would like to use that source code, which is no longer supported by Esri, go to [the 10.2 release](https://github.com/Esri/message-simulator-qt/releases/tag/v10.2) or [the last commit that included Vehicle Commander](https://github.com/Esri/message-simulator-qt/tree/ee7b931565859b34c77d493d4c1333c2424d29e3). It is recommended that you use the current [Vehicle Commander](https://github.com/Esri/vehicle-commander-java) instead.
+
+![Image of Message Simulator]( ScreenShot.jpg "message-simulator-qt")
 
 ## Features
 
-* Displays high-performance touchscreen maps, including provisioned basemaps and operational data
-* Displays a GPS location (simulated)
-* Displays military symbology and grids
+* Takes GeoMessage XML files as input
+* Writes UDP datagrams as output
```

Comment:
Might want to mention that these are "broadcast" datagrams - since this has some network/security ramifications